frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupItem/groupItem.component.tsx

```diff
@@ -60,6 +60,8 @@ export const GroupItem = ({ override, index }: GroupProps) => {
 	const isAdmin = ProjectsHooksSelectors.selectIsProjectAdmin();
 	const { group, color, opacity } = override;
 	const checked = getGroupIsChecked(index);
+	const isHidden = groupType === 'hidden';
```

Comment:
When editing an existing hidden group it the group stays selected showing the same problem
![show hidden group](https://github.com/3drepo/3drepo.io/assets/42032433/bbd27c1a-bd0f-4ae7-a0d0-fd0837694c4b)